workflow/scripts/osemosys_global/powerplant/renewable_targets.py

```diff
@@ -0,0 +1,212 @@
+"""Function to set renewable targets."""
+import pandas as pd
+import itertools
+
+from data import get_years
+
+def apply_re_pct_targets(re_targets, remove_nodes, oar_df, renewables_list, 
+                     fuel_set, annual_demand_df, region_name):
```

Comment:
This is a cool way to do the renewable generation target! I wouldnt have thought to do it through `AccumulatedAnnualDemand`, and its much cleaner than any idea I had!